.github/workflows/typos.yaml

```diff
@@ -0,0 +1,14 @@
+name: Spell Check Action
+on: [pull_request]
+
+jobs:
+  typos-check:
+    name: Spell Check with Typos
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Actions Repository
+        uses: actions/checkout@v3
+      - name: Check spelling with custom config file
+        uses: crate-ci/typos@v1.14.8
+        with:
+          config: ./typos.toml
```

Comment:
Not neccessary, install it in editor may be better.

Reply:
Why not add it to CI?

Reply:
In My Humble Opinion IMO. Less is better, there is no guarantee that anything will go wrong^-^. And it may change the code?

Reply:
But CI is necessary. Then what do you want to add to CI?

Reply:
I think e2e is import but it has not yet considered whether this is feasible and difficult.